pages/participantinfo.tsx

```diff
@@ -64,337 +75,918 @@ export default function ParticipantInfo({
-        "Parent Guardian Information",
         "Participant Emergency Form",
         "Participant Health Form",
+        "Parent Guardian Information",
         "Confirm Participants",
         "Proof of Income",
         "How did you hear about us?",
     ];
 
     const formPages = [
-        <FormPage>
-            <Box maxW="55rem">
-                <Text noOfLines={2} fontSize="16px" fontWeight="200">
-                    Please provide information on the participant that is being
-                    registered in the program. An opportunity to add information
-                    of additional participants you would like to register will
-                    be provided afterwards.
-                </Text>
-            </Box>
-            <FormLabel>
-                Participant Name
+        <Box>
+            <FormPage>
+                <Box maxW="55rem">
+                    <Text noOfLines={2} fontSize="16px" fontWeight="200">
+                        Please provide information on the participant that is
+                        being registered in the program. An opportunity to add
+                        information of additional participants you would like to
+                        register will be provided afterwards.
+                    </Text>
+                </Box>
+                <FormLabel>
+                    Participant Name
+                    <HStack spacing="24px">
+                        <FormControl id="participant-first-name">
+                            <Input
+                                placeholder="First name"
+                                onChange={(e) =>
+                                    setParticipantFirstName(e.target.value)
+                                }
+                                value={participantFirstName}
+                            />
+                        </FormControl>
+                        <FormControl id="participant-last-name">
+                            <Input
+                                placeholder="Last name"
+                                onChange={(e) =>
+                                    setParticipantLastName(e.target.value)
+                                }
+                                value={participantLastName}
+                            />
+                        </FormControl>
+                    </HStack>
+                </FormLabel>
+                <FormControl id="date-of-birth">
+                    <FormLabel>Date Of Birth (YYYY-MM-DD) </FormLabel>
+                    <Input
+                        placeholder="Date Of Birth"
+                        onChange={(e) => setDateOfBirth(e.target.value)}
+                        value={dateOfBirth}
+                    />
+                </FormControl>
+                <FormControl id="street-address-1">
+                    <FormLabel>Street Address 1</FormLabel>
+                    <Input
+                        placeholder="249 Phillip Street"
```

Comment:
Let's use this address as the placeholder values

1924 W 45th Ave, Vancouver, BC V6M 3S3

It's just a random elementary school I found in Vancouver lol

Reply:
sure, or do you want me to use the actual SDC address?